### ISSUE TITLE
Add support for plotly.js

### ISSUE THREAD

**Issue by timvink** [timvink/mkdocs-charts-plugin#9]
Could be cool to add and probably relatively low effort.

**Comment by timvink** [comment]
Reading in data files is slightly more involved though: https://plotly.com/javascript/ajax-call/ 
That also means support for `data_path` will be trickier. Actually the entire syntax seems more contrived because javascript doesn't have the concept of dataframes.

Instead, we could use plotly python. 

```python
import plotly.express as px
long_df = px.data.medals_long()

fig = px.bar(long_df, x="nation", y="count", color="medal", title="Long-Form Input")
fig.show()
```

we can drop the import, and the `fig.show()`. 

The data loading is trickier to solve, as the `fence_plotly` function wouldn't know about the project root.

**Comment by timvink** [comment]
So back to plotly.js. 

If we require data to be stored a `.json` files, we can ask user to point to it.

```javascript
var plot_data = custom_load_data_function("...")

var trace1 = {
  x: plot_data['x'],
  y: plot_data['y'],
  type: 'scatter',
};

var data = [trace1];
```

For for each block we can run the javascript and 

```
Plotly.newPlot('myDiv', data, {}, {showSendToCloud: true});
```

https://stackoverflow.com/questions/58576662/django-plotly-js-pass-pandas-dataframe-columns-to-template-chart

**Comment by haoda-li** [comment]
Hello, I've actively used plotly and mkdocs for creating notebooks. Thanks for this great work! 

I got innovated by this repo and implemented a Plotly charts version. The mechanism of mine is a bit different. I left the URL updates in mkdocs' `on_page_content` stage. Therefore, I did not bother to send a pull request and merge with your implementation. 

Feel free to check my implementation here and give some suggestions!  
https://github.com/haoda-li/mkdocs-plotly-plugin
